#! /bin/sh /usr/share/dpatch/dpatch-run
## 357189_gcc4.1_ftbfs.dpatch by Martin Michlmayr <tbm@cyrius.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix #357189: GCC 4.1 FTBFS

@DPATCH@
diff -urNad gdal-1.3.1~/frmts/fit/fitdataset.cpp gdal-1.3.1/frmts/fit/fitdataset.cpp
--- gdal-1.3.1~/frmts/fit/fitdataset.cpp	2005-07-06 01:10:52.000000000 +0800
+++ gdal-1.3.1/frmts/fit/fitdataset.cpp	2006-03-16 13:40:47.000000000 +0800
@@ -174,7 +174,7 @@
 
 public:
 
-    FITRasterBand::FITRasterBand( FITDataset *, int );
+    FITRasterBand( FITDataset *, int );
 	~FITRasterBand();
     
     // should override RasterIO eventually.
